src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -108,6 +108,28 @@ public without sharing class UTIL_OrgTelemetry_SVC {
         Database.executeBatch(new UTIL_OrgTelemetry_BATCH(), 1);
     }
 
+    /**
+     * @description Determines if we should execute the telemetry code based on if the created custom metdata types
+     * are NPSP metadata type objects. 
+     * @param customMetadataTypeSet
+     */
+    @Future
+    public static void asyncProcessCMTChange(Set<String> customMetadataTypeSet) {
+
```

Comment:
@patrick-yan-sf  I Just updated the code, is this more in line with with you were thinking? I no longer do a split on a period, instead I just get the whole name and use the enum values to see if it's contained.

Reply:
@KietTruongSFDO - Thanks.  Yes that's what I was thinking of.